exercises/practice/killer-sudoku-helper/src/test/java/KillerSudokuHelperTest.java

```diff
@@ -0,0 +1,57 @@
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class KillerSudokuHelperTest {
+
+    private KillerSudokuHelper helper = new KillerSudokuHelper();
```

Comment:
To help catch issues in student implementations where they store state in the class instance, it might be better to create a new instance for every test using a setup method. What do you think?